exec.go

```diff
@@ -91,7 +111,83 @@ EXAMPLE:
 		},
 	},
 	Action: func(context *cli.Context) error {
-		// TODO
-		return nil
+		params := execParams{
+			cID: context.String("container-id"),
+			console: context.String("console"),
+			cwd: context.String("cwd"),
+			envs: context.String("env"),
+			tty: context.String("tty"),
+			user: context.String("user"),
+			process: context.String("process"),
+			detach: context.String("detach"),
+			pidFile: context.String("pid-file"),
+			processLabel: context.String("process-label"),
+			apparmor: context.String("apparmor"),
+			noNewPrivs: context.String("no-new-privs")),
+			caps: context.String("cap")),
+			noSubreaper: context.String("no-subreaper")),
+		}
+
+		return exec(params)
 	},
 }
+
+func create(params execParams) error {
+	if err := validContainer(params.cID); err != nil {
+		return err
+	}
+
+	podStatus, err := vc.StatusPod(params.cID)
+	if err != nil {
+		return err
+	}
+
+	if len(podStatus.ContainersStatus) != 1 {
+		return fmt.Errorf("ContainerStatus list from PodStatus %s is wrong, expecting only one ContainerStatus", containerID)
```

Comment:
Show statuses on error? Also, this feels like it might need a `BUG` prefix as this situation indicates a programming error rather than user error / resource availability issue I think?